providers/dns/cloudflare/cloudflare.toml

```diff
@@ -2,7 +2,7 @@ Name = "Cloudflare"
 Description = ''''''
 URL = "https://www.cloudflare.com/dns/"
 Code = "cloudflare"
-Since = "v0.3.0"
+Since = "v0.3.0 (API tokens since v3.1.0)"
```

Comment:
Could you move `(API tokens since v3.1.0)` inside the env vars description.

Reply:
Done.